db/migrate/20240424175612_add_team_id_to_api_keys.rb

```diff
@@ -0,0 +1,5 @@
+class AddTeamIdToApiKeys < ActiveRecord::Migration[6.1]
+  def change
+    add_column :api_keys, :team_id, :integer
```

Comment:
You can add this migration to the above one `AddTitleDescUserIdToApiKeys` as we still in same PR